### PR TITLE
API v3

### PR DESCRIPTION
**Pull request by VDmitryO**

### CODE REVIEW

File: lib/be_gateway/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module BeGateway
-  VERSION = '1.0.3'
+  VERSION = '1.1.0'
```
Comment (obmax) [inline lib/be_gateway/version.rb:2]: please don't change version by yourself

File: README.md [hunk 2]
```diff
@@ -179,6 +179,20 @@ response.errors["amount"] # => ["must be an integer"]
 response.errors["recipient_card"]  # => { "number" => ["is not a card number"] }
 ```
 
+### APIv3
```
Comment (obmax) [inline README.md:182]: let's add to readme example of making v3 request

File: lib/be_gateway/connection.rb [hunk 3]
```diff
@@ -76,6 +97,13 @@ def connection
       end
     end
 
+    def headers
+      {}.tap do |h|
```
Comment (obmax) [inline lib/be_gateway/connection.rb:101]: (passed_headers || {}).tap do |h|

File: lib/be_gateway/version.rb (hunk 1 above)
Comment (VDmitryO) [reply lib/be_gateway/version.rb:2]: It's okey, I discussed this with Artem

**Review by obmax — APPROVED** [review]

**Pull request merged**